hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/network/UpgradeActions.java

```diff
@@ -151,7 +151,12 @@ private CompletableFuture<Void> extractNow(
         return runAsync(
                 () -> {
                     try {
-                        FileUtils.cleanDirectory(new File(artifactsLoc));
+                        var artifactsLocFile = new File(artifactsLoc);
+                        if (artifactsLocFile.exists()) {
+                            FileUtils.cleanDirectory(new File(artifactsLoc));
```

Comment:
should you be using `FileUtils.cleanDirectory(artifactsLocFile);`

Reply:
good catch!